packages/syntax-icons/src/Icons.stories.tsx

```diff
@@ -0,0 +1,370 @@
+  { name: "HeartUnfilled", component: HeartUnfilled },
+  { name: "Help", component: Help },
+  { name: "Home", component: Home },
+  { name: "IdentifyDocument", component: IdentifyDocument },
+  { name: "Information", component: Information },
+  { name: "Joystick", component: Joystick },
+  { name: "Keyboard", component: Keyboard },
+  { name: "Leaf", component: Leaf },
+  { name: "Lightbulb", component: Lightbulb },
+  { name: "List", component: List },
+  { name: "Location", component: Location },
+  { name: "Menu", component: Menu },
+  { name: "Message", component: Message },
+  { name: "MicrophoneOff", component: MicrophoneOff },
+  { name: "MicrophoneOn", component: MicrophoneOn },
+  { name: "More", component: More },
+  { name: "MultiMediaDocument", component: MultiMediaDocument },
+  { name: "MultipleUsers", component: MultipleUsers },
+  { name: "NewStarter", component: NewStarter },
+  { name: "Pause", component: Pause },
+  { name: "Pencil", component: Pencil },
+  { name: "PhoneOff", component: PhoneOff },
+  { name: "PhoneOn", component: PhoneOn },
+  { name: "Play", component: Play },
+  { name: "Progress", component: Progress },
+  { name: "Refresh", component: Refresh },
+  { name: "Safety", component: Safety },
+  { name: "Screen", component: Screen },
+  { name: "Search", component: Search },
+  { name: "SearchUsers", component: SearchUsers },
+  { name: "Send", component: Send },
+  { name: "Settings", component: Settings },
+  { name: "Shop", component: Shop },
+  { name: "Shuffle", component: Shuffle },
+  { name: "SideArrow", component: SideArrow },
+  { name: "SquareEnclosedPlus", component: SquareEnclosedPlus },
+  { name: "StarFilled", component: StarFilled },
+  { name: "StarHalfFilled", component: StarHalfFilled },
+  { name: "Stars", component: Stars },
+  { name: "StarUnfilled", component: StarUnfilled },
+  { name: "Subtitles", component: Subtitles },
+  { name: "Suitcase", component: Suitcase },
+  { name: "SuperTutor", component: SuperTutor },
+  { name: "Sync", component: Sync },
+  { name: "TeachingCertificate", component: TeachingCertificate },
+  { name: "TextDocument", component: TextDocument },
+  { name: "ThumbsDown", component: ThumbsDown },
+  { name: "ThumbsUp", component: ThumbsUp },
+  { name: "Timer", component: Timer },
+  { name: "ToggleSettings", component: ToggleSettings },
+  { name: "Translate", component: Translate },
+  { name: "TwoUsers", component: TwoUsers },
+  { name: "Type", component: Type },
+  { name: "User", component: User },
+  { name: "Wifi", component: Wifi },
+  { name: "WifiRouter", component: WifiRouter },
+];
+
+const MUIIcons = [{ name: "Repeat", component: Repeat }];
+
+export default {
+  title: "Icons/Icons",
+  component: Icon,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/%F0%9F%93%90-Syntax?node-id=1206-4420&t=yFh7Ijhf6PU7Lin3-0",
+    },
+  },
+  argTypes: {
```

Comment:
@somethiiing can you also add `args` to set the defaults? See other stories for examples

Reply:
https://github.com/Cambly/syntax/blob/660ee7f8885ebbfc2f9684e6302ff429e87ec728/packages/syntax-core/src/Badge/Badge.stories.tsx#L15-L17